docs/source/usersguide/processing.rst

```diff
@@ -103,18 +103,119 @@ The code snippet below illustrates one possible configuration of parameters to t
 
 .. code-block:: python
 
-    import openmoc.process as proc
+    import openmoc.process
 
     # Retrieve the simulation state(s) stored in the 'states.h5' file
     # and returns the data in a Python dictionary
-    simulation_state = proc.restore_simulation_state(filename='states.h5')
+    simulation_state = openmoc.process.restore_simulation_state(filename='states.h5')
 
 
------------------------
-Computing Fission Rates
------------------------
+------------------------
+Computing Reaction Rates
+------------------------
 
-In some cases, a user may wish to only compute and export the fission rates in each flat source region for a simulation. In this case, the ``compute_fission_rates(...)`` routine in the ``openmoc.process`` module  may be used. The routine takes in a ``Solver`` subclass (e.g., ``CPUSolver``, ``VectorizedSolver``, ``GPUSolver``, etc.) and computes the fission rate for each ``Universe`` in the ``Geometry`` by summing up the fission rates in each ``Cell`` in the ``Universe``. In most cases, a ``Universe`` is replicated in many places throughout the ``Geometry``. To account for this, the routine will separately compute the fission rates for each unique placement of that ``Universe`` in the ``Geometry``. By default, the fission rates will be exported to a Python pickle_ file, but may alternatively be exported to an HDF5_ binary file. Each fission rate will be indexed by a string representing the "path" of ``Universes``, ``Lattices`` and ``Lattice`` cell indices traversed through the ``Geometry`` to reach the flat source region of interest. :ref:`Table 4 <table_fission_rates>` describes the parameters accepted by the routine.
+
+Reaction Rates By Tally Mesh
+----------------------------
+
+The ``openmoc.process`` module supports reaction rate through structured Cartesian mesh tallies, similar to how one might compute such quantities in a Monte Carlo code such as MCNP, Serpent or OpenMC. The ``Mesh`` class allows one to tally a reaction rate in each of its structured cells. :ref:`Table 4 <table_mesh_params>` below itemizes each of the property attributes of the ``Mesh`` class which can be used to tally reaction rates.
+
+.. _table_mesh_params:
+
+========================  ===================  ==========  ==========================================================
+Property                  Type                 Default     Note
+========================  ===================  ==========  ==========================================================
+``dimension``             list of integers     None        The number of mesh cells in each direction
+``lower_left``            list of floats       None        The lower-left corner of the structured mesh
+``upper_right``           list of floats       None        The upper-right corner of the structrued mesh
```

Comment:
The reason I went with `lower_left` and `upper_right` was to mirror the `openmc.Mesh` class as much as possible. At least from my perspective, this mesh tally feature is intended to make it very easy to compare spatially-integrated reaction rates between the two codes. The way it is implemented now, it is very easy to create an `openmoc.process.Mesh` from an `openmc.mesh`:

```python
...

# Extract mesh for mesh tallies from OpenMC StatePoint
openmc_mesh = statepoint.meshes[mesh_id]

# Create OpenMOC Mesh to mirror OpenMC Mesh
openmoc_mesh = openmoc.process.Mesh()
openmoc_mesh.dimension = openmc_mesh.dimension
openmoc_mesh.lower_left = openmc_mesh.lower_left
openmoc_mesh.upper_right = openmc_mesh.upper_right
openmoc_mesh.width = openmc_mesh.width

...
```

I think we should eitheradopt a standard akin to our own `Lattice` or `openmc.Mesh`. Based on the use case, I would go with the latter since it simplifies the code above (though I recognize that `lower_left` and `upper_right` can easily be backed out of `origin` and `offset` and vice versa).